projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/RecommendationsPage.svelte

```diff
@@ -37,12 +50,26 @@
 	<Header subPage={__( 'Image analysis report', 'jetpack-boost' )} />
 	<div class="recommendations-page jb-section--alt">
 		<div class="jb-container">
-			<Hero {needsRefresh} {refresh} />
-			<Tabs />
+			<Hero
+				{needsRefresh}
+				{refresh}
+				{isImageCdnModuleActive}
+				{isaLastUpdated}
+				{hasActiveGroup}
+				totalIssueCount={$totalIssueCount}
```

Comment:
We'll eventually get rid of global variables everywhere, might be a good idea to group this with the rest

Reply:
I'm not sure I understand what you mean by "group this with the rest" 😅

Reply:
Oh, sorry, I meant that `totalIssueCount={$totalIssueCount}` could become `totalIssueCount={issueCount}` and instead define on top of the file that `$: issueCount = $totalIssueCount` or something along those lines - so that global store subscriptions aren't present in the template side of things - it's a minor thing, but I think it will help conversion to react later.

Reply:
Thanks! Updated in 6fbd59d.